pages/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { GetServerSideProps } from 'next';
+
+export const getServerSideProps: GetServerSideProps = async () => {
```

Comment:
ここも静的エクスポートと併用することができないので、以下のようにしたいです。
```suggestion
export const getStaticProps: GetStaticProps = async () => {
```

Reply:
こちらで修正しました！

https://github.com/Komatsuzaki-Sogo/next-contentful/pull/1#discussion_r2137315053